### PR TITLE
Configure Travis CI

### PR DESCRIPTION
**Pull request by Tomatosoup97**

### CODE REVIEW

File: run_tests.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    print('Run python tests')
+    do_call([
+        'sh', '-c',
+        'python -m unittest',
+    ])
+
+
+def main():
+    run_python_tests()
+    run_flake8()
```
Comment (mlazowik) [inline run_tests.py]: I'd run linters first, as projects get bigger they run faster than tests.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+    - 3.4
+    - 3.5
+    - 3.6
+script:
+    - python -m unittest
```
Comment (mlazowik) [inline .travis.yml]: Don't you want this to be `python run_tests.py`?

Comment (Tomatosoup97) [reply .travis.yml]: FIxed

File: run_tests.py (hunk 1 above)
Comment (Tomatosoup97) [reply run_tests.py]: Done

**Pull request merged**